### DISCUSSION TITLE
Media Library with resize capabilities

### DISCUSSION THREAD

**Discussion by kapik** [strapi/strapi#5898] [Legacy Topics]
Hey !

You have released the media library but the resize feature mentioned throw the [roadmap](https://portal.productboard.com/strapi/1-public-roadmap/c/28-media-library-with-edit-crop-resize-capabilities) is missing 😢 

Do you plan to add it 🙏 ?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Reply by kapik** [reply]
Sorry, I didn't know this feature !

**Comment by Aurelsicoko** [comment]
We provide an easy way to crop and what we call the "Responsive-friendly" feature which resizes the uploaded picture. When you upload a picture, you can access it using the generated URL, but by adding the following prefixes (large_, medium_, small_, thumbnail_) you can access a smaller version of the picture.

`http://localhost:1337/uploads/mike-meyers.jpeg`
`http://localhost:1337/uploads/large_mike-meyers.jpeg`
`http://localhost:1337/uploads/medium_mike-meyers.jpeg`
`http://localhost:1337/uploads/small_mike-meyers.jpeg`
`http://localhost:1337/uploads/thumbnail_mike-meyers.jpeg`

I hope it will help :)

**Reply by DarkTraveler** [reply]
@Aurelsicoko Hey. Can I crank this up with already uploaded images? Before upgrading to version 20, 600 images had already been downloaded and did not work with them.

**Reply by julesrenaud** [reply]
Hi! I'm puzzled as to why there isn't a way to define responsive image sizes. Is there a plan to make it so we can define width and height of responsive image sizes?

**Reply by derrickmehaffy** [reply]
@julesrenaud I will pass this along to our product design team and see if they had any plans in the works, I can imagine eventually yes we will want users to be able to customize this, but I am not aware of any current plans. If possible can you open up a new Feature Request on the Github issues and on our Product Board (there is a link in the Feature Request template)

**Reply by julesrenaud** [reply]
> @julesrenaud I will pass this along to our product design team and see if they had any plans in the works, I can imagine eventually yes we will want users to be able to customize this, but I am not aware of any current plans. If possible can you open up a new Feature Request on the Github issues and on our Product Board (there is a link in the Feature Request template)

Thanks for the quick answer! I will go ahead and propose the feature in gh issues and on the product board
